Classes/Core/EidRequestBootstrap.php

```diff
@@ -63,10 +63,9 @@ public function __construct() {
 		$typoScriptFrontendController->fe_user = $feUserObj;
 		$typoScriptFrontendController->id = $pageId;
 		$typoScriptFrontendController->determineId();
-		$typoScriptFrontendController->getCompressedTCarray();
 		$typoScriptFrontendController->initTemplate();
 		$typoScriptFrontendController->getConfigArray();
-		$typoScriptFrontendController->includeTCA();
+		EidUtility::initTCA();
```

Comment:
Ha, ha. A perfect demonstration, why it is a bad idea to use such eID dispatchers ;)

Thanks for figuring out a solution which works in both TYPO3 versions!